lib/internal/main/repl.js

```diff
@@ -37,6 +36,7 @@ if (process.env.NODE_REPL_EXTERNAL_MODULE) {
     process.exit(kGenericUserError);
   }
 
+  const esmLoader = require('internal/process/esm_loader');
```

Comment:
We would typically store the value locally to save the call to `require` on the following calls, not sure how relevant it is but for consistency sake, I think we should do it.
```suggestion
  esmLoader ??= require('internal/process/esm_loader');
```

Reply:
It's only ever called once during startup, it should therefore be fine, no?